backend/accounts.go

```diff
@@ -969,7 +967,7 @@ type scriptTypeWithKeypath struct {
 	keypath    signing.AbsoluteKeypath
 }
 
-// adds a combined BTC account with the given script types.
+// adds a combined BTC account with the given script accountsTypes.
```

Comment:
wrong search&replace

Reply:
whoops!

Reply:
Fixed together with a few others, thanks for catching it!